### ISSUE TITLE
Start draging with menu open

### ISSUE THREAD

**Issue by meijer3** [jonataswalker/ol-contextmenu#263]
Hi,

As UX point of view, I would like to be able to start drag/pan even with the menu open.
Now: open menu > drag outside > menu closes (and hightlight the page...)
Desired: open menu > direct panable (outside the menu ofcourse, and it may close)

![2023-04-14 09-00-31](https://user-images.githubusercontent.com/45625341/231968215-4d84d5c5-da23-41b7-a87c-10f49cddb0c9.gif)

**Comment by jonataswalker** [comment]
Not clear to me how this "it may close" would work.

Are you able/willing to provide a PR?

**Comment by GastonZalba** [comment]
I agree with that behavior and i can make a PR for this, but some questions:

Removing the line [307](https://github.com/jonataswalker/ol-contextmenu/blob/master/src/main.ts#L307) will allow the popup to be closed on every click (dragging or not) and the event will propagate to the map (allowing dragging it... and maybe select a feature?). If this is ok, the fix it's quite easy.

If instead has to close and propagate only while dragging, i think the listener at [303](https://github.com/jonataswalker/ol-contextmenu/blob/master/src/main.ts#L303) could be replaced with a map event (a `singleclick`), instead of the generic  `pointerdown`. With the ol event it's easy to check the dragging event and allow the propagation only in that case. The only caveat with this is that the `singleclick` is delayed by 250 ms, so it will no be closed inmediatly.

**Comment by JakobMiksch** [comment]
I find this issue important as well.

**Comment by ztolley** [comment]
I have a similar ask, though in my case the map is tracking an item, when the item moves the map centre co-ordinates are updated to the target position. This seems to trigger the menu to close. Maybe we could add an optional argument defaulted to true named something like 'hideOnMove'.  I'll try this on a local copy
